terraform/modules/bosh_vpc/network_private.tf

```diff
@@ -100,3 +98,13 @@ resource "aws_instance" "az2_private_nat" {
     Name = "${var.stack_description}  (AZ2 NAT)"
   }
 }
+
+resource "aws_eip" "az1_nat_eip" {
```

Comment:
Should probably add  `lifecycle {prevent_destroy = true}` to these resources, as these are going to end up in other people's firewall configurations and if one gets deleted on accident, there's no getting that same ip back from AWS again